website/static/js/markdown.js

```diff
@@ -68,6 +92,7 @@ var markdown = new MarkdownIt('commonmark', {
 var markdownQuick = new MarkdownIt('commonmark', { linkify: true })
     .use(require('markdown-it-sanitizer'))
     .use(require('markdown-it-imsize'))
+    .use(viewOnlyImage)
```

Comment:
Nit: I'm not sure this plugin is necessary for `markdownQuick`. We only use `markdownQuick` while a user is editing the wiki, in which case they will already have read access to the images. But maybe it's better to have close parity between markdownQuick and markdownFull 🤔  . No big deal: this is fine as is.